### PR TITLE
update jobs/2015-11-25-Frontend-Dev-for-Jolocom.md

### PR DESCRIPTION
**Pull request by Joachim16**
Fronted Developer - Jolocom

### CODE REVIEW

File: jobs/2015-11-25-Frontend-Dev-for-Jolocom.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+---
+layout: jobs
+title: FRONTEND DEVELOPER / DESIGNER
+role: Job Role # Eg: front-end, ux design, data visualization, javsacript, CSS, HTML, D3js, React
```
Comment (evalica) [inline jobs/2015-11-25-Frontend-Dev-for-Jolocom.md]: typo: javascript

File: jobs/2015-11-25-Frontend-Dev-for-Jolocom.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+2. Some prototyping software/service could come in handy too, like UXPin
+
+Tech skills:
+1. Skilled in JavaScript (even without jQuery)
+2. Knowledge of at least one major JS framework: React, Ember, Aurelia, Backbone, Angular, Vue, etc
+3. D3sj is a plus
+4. Experience with CSS concepts: BEM, OOCSS, etc; good understanding of how specificity works
+5. Experience with pre-processors, either SASS or LESS would do
+6. Some experience with task runners Grunt/Gulp (minification, auto-prefixing)
+(7). Some knowledge of flexbox
```
Comment (evalica) [inline jobs/2015-11-25-Frontend-Dev-for-Jolocom.md]: Additional parenthesis

File: jobs/2015-11-25-Frontend-Dev-for-Jolocom.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
+---
+layout: jobs
+title: FRONTEND DEVELOPER / DESIGNER
+role: Job Role # Eg: front-end, ux design, data visualization, javsacript, CSS, HTML, D3js, React
+organization: Jolocom
+github: https://github.com/jolocom
+contact: joachim@jolocom.com
+contributing_md: (optional) 
+contributors_md: (optional)
```
Comment (jancborchardt) [inline jobs/2015-11-25-Frontend-Dev-for-Jolocom.md]: I guess both of these can be removed so they don’t show up, right? @simonv3

File: jobs/2015-11-25-Frontend-Dev-for-Jolocom.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
+---
+layout: jobs
+title: FRONTEND DEVELOPER / DESIGNER
```
Comment (jancborchardt) [inline jobs/2015-11-25-Frontend-Dev-for-Jolocom.md]: Normal case (not all caps) would be cool. ;)

File: jobs/2015-11-25-Frontend-Dev-for-Jolocom.md (hunk 3 above)
Comment (simonv3) [inline jobs/2015-11-25-Frontend-Dev-for-Jolocom.md]: Do you mean in this instance or in the original file?

**Comment by gillisig** [comment]
@Joachim16 Do you still want this merged? I guess it kind of slipped through the cracks, sorry about that.

**Comment by Joachim16** [comment]
yes, would be great!

**Comment by simonv3** [comment]
Thanks @gillisig for going through things!

**Pull request merged**